app/mailers/collections_mailer.rb

```diff
@@ -48,7 +48,23 @@ def review_access_removed_email
       'collection in the SDR')
   end
 
-  def collection_activity
+  def new_first_draft_created
```

Comment:
would "first draft created" be sufficient, as "created" implies new?   "updated" would imply changes ...